albogdan/skule-vote
# Fixed position to election

## Overview

- Resolves #57 
- Forgot to change a `Position` to `Election` in a previous PR. This fixes that

## Unit Tests Created

- N/A


## Steps to QA

- First once you pull do the following to apply the migration:
   - `python manage.py migrate backend zero`
   - `python manage.py migrate backend`
- Go to the admin site and try to create an `Election`. Enter `seats_available` as 0.
   - You should see the message: `There must be at least one Election open.`
